### PR TITLE
Complete school profile and login component

### PR DESCRIPTION
**Pull request by yirano**
- School profile has been styled
- School form populates with data to edit
- School profile renders data
- Login Component is styled, hooked up to state management, and ready for endpoints.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Jahteo) [inline .vscode/settings.json:1]: what's this?

**Review by Nicci498 — APPROVED** [review]

File: src/components/pages/Headmaster/HeadmasterDashboard.js [hunk 2]
```diff
@@ -65,25 +65,20 @@ function HeadmasterDashboard() {
   return (
     <div>
       <div className={desktop ? 'headmasterDashboard' : null}>
-        <h1>Hello, Headmaster!</h1>
         <Switch>
-          <Route path="/headmaster/mentor-pairings" component={TestComponent} />
-          <Route path="/headmaster/mentor-advisor" />
-          <Route path="/headmaster/school-village">
+          <Route path="/mentor-pairings" component={TestComponent} />
```
Comment (Jahteo) [inline src/components/pages/Headmaster/HeadmasterDashboard.js:69]: Project Manager's gonna be soo happy to have these shorter urls!

**Review by Jahteo — APPROVED** [review]
Looks good

File: .vscode/settings.json (hunk 1 above)
Comment (yirano) [reply .vscode/settings.json:1]: It's a workspace setting the VSCode sets up I think...

**Pull request merged**